src/components/types/Date.css

```diff
@@ -0,0 +1,217 @@
+.rdt {
```

Comment:
It's pretty hard to load CSS specific to our manager's code at Storybook.
So, we need to think something else.

Here's something we can do.
See this library: https://github.com/substack/insert-css

We need to write this file as a `.js` file which returns a big string of CSS.
Then we can simply use above library to apply CSS.